packages/common/env/package.json

```diff
@@ -21,10 +20,11 @@
   },
   "peerDependencies": {
     "@affine/templates": "workspace:*",
-    "@blocksuite/global": "0.0.0-20230409084303-221991d4-nightly"
```

Comment:
This was making it impossible for Reflame to resolve `@blocksuite/global` to a valid version, since every package that uses this has a strict dependency on version `0.11.0-nightly-202401020419-752a5b8`.

Reflame's npm package resolution algorithm is based on NPM, which treats this as an error, while yarn only outputs a warning.